docs/src/main/paradox/project/migration-guides.md

```diff
@@ -10,16 +10,29 @@ please [migrate to Akka 2.6](https://doc.akka.io/docs/akka/current/project/migra
 
 This is just stub documentation. It will be improved.
 
-* for Pekko jar dependencies, the groupId is "org.apache.pekko" instead of "com.typesafe.akka"
-* the jar names start with "pekko" instead of "akka" - e.g. pekko-actor_2.13.jar instead of akka-actor_2.13.jar 
+In your dependency management files:
+* Update dependency Akka version to the latest version of Pekko `1.0.x`.
+* for Pekko jar dependencies, the groupId is `org.apache.pekko` instead of `com.typesafe.akka` or `com.lightbend.akka.<package_name>`
+* the jar names start with "pekko" instead of "akka" - e.g. pekko-actor_2.13.jar instead of akka-actor_2.13.jar
 * Alpakka equivalent is "pekko-connectors" - e.g. pekko-connectors-kafka_2.13.jar instead of alpakka-kafka_2.13.jar
-* Pekko packages start with "org.apache.pekko" instead of "akka" - e.g. `import org.apache.pekko.actor` instead of `import akka.actor`
-* Where class names have "Akka" in the name, the Pekko ones have "Pekko" - e.g. PekkoException instead of AkkaException
-* Configs in `application.conf` use "pekko" prefix instead of "akka"
+
+In config files:
+* Configs use `pekko` prefix instead of `akka`
```

Comment:
why remove the reference to 'application.conf' ?

Reply:
To be more accurate we could use `*.conf` instead since some conf files are `reference.conf` and others are `applciation.conf`.

in fact from checking the codebase it should all be `reference.conf`, after all the idea is that `reference.conf` is whats used for libraries and `application.conf` is whats used for applications.

Reply:
it's a migration guide for users migrating their own code bases - they will be updating their own application.conf files

some lib maintainers might have reference.conf files

I suppose, some users use '.json' files for configs as well (because typesafe config supports those too). 

I would still prefer if this section, tries to name some of the more common config files to help out users to know what to look for - imagaine if the Akka based app that is being migrated was written 5 years ago and the original writers have moved on.

Reply:
> it's a migration guide for users migrating their own code bases - they will be updating their own application.conf files

Right but its also relevant to libraries which integrate with Pekko, in which case they would be dealing with `reference.conf` files and not `application.conf`. Not all users are applications.

> I suppose, some users use '.json' files for configs as well (because typesafe config supports those too).

And also `*.prop` files too which afaik are compatible with HOCON format.

> I would still prefer if this section, tries to name some of the more common config files to help out users to know what to look for - imagaine if the Akka based app that is being migrated was written 5 years ago and the original writers have moved on.

In that case the easiest thing would be to search for `akka` prefix in files that are known to hold configurations which would be `*.conf`/`*.prop` and `*.json`.